src/main/kotlin/io/openfuture/state/config/ProcessingConfig.kt

```diff
@@ -0,0 +1,15 @@
+package io.openfuture.state.config
+
+import io.openfuture.state.property.WatcherProperties
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class ProcessingConfig(private val watcherProperties: WatcherProperties) {
```

Comment:
Well. As properties are `Watcher` properties, does it make sense to have `WatcherConfig` as well?